src/robot.coffee

```diff
@@ -269,6 +269,8 @@ class Robot
     user    = process.env.EXPRESS_USER
     pass    = process.env.EXPRESS_PASSWORD
     stat    = process.env.EXPRESS_STATIC
+    port    = process.env.EXPRESS_PORT or process.env.PORT or 8080
+    address = process.env.BIND_ADDRESS or '0.0.0.0'
```

Comment:
I don't see any reason not to do the same namespacing with BIND_ADDRESS. Leaving it as the lone non-namespaced variable for Express just seems strange.